source/funkin/play/PlayState.hx

```diff
@@ -1477,7 +1477,9 @@ class PlayState extends MusicBeatSubState
       }
 
       if (!startingSong
-        && (Math.abs(FlxG.sound.music.time - correctSync) > 5 || Math.abs(playerVoicesError) > 5 || Math.abs(opponentVoicesError) > 5))
+        && (Math.abs(FlxG.sound.music.time - correctSync) > 15
+          || Math.abs(playerVoicesError) > 15
+          || Math.abs(opponentVoicesError) > 15)) // 5ms was too little according to the command prompt that says it should be 15ms, my bad gang!
```

Comment:
Tiny nitpick but this comment is not really needed.

Side note: I really hope this is tested by other people so it's fixed for real this time.